src/js/transports/native.ts

```diff
@@ -8,10 +10,23 @@ export class NativeTransport implements Transport {
   /** A simple buffer holding all requests. */
   protected readonly _buffer: PromiseBuffer<Response> = new PromiseBuffer(30);
 
+  /** Hack that's used to send transactions on Android */
+  private _fetchTransport: Transport;
```

Comment:
you can now experiment with https://github.com/getsentry/sentry-java/releases/tag/4.0.0-alpha.3
that should resolve the problem and the hack could be removed

Reply:
Ok thanks, I'll give it a try and I think we can have an alpha ready by like Monday.